crates/sovereign-sdk/full-node/db/sov-schema-db/src/lib.rs

```diff
@@ -65,6 +66,7 @@ impl DB {
             column_families.into_iter().map(|cf_name| {
                 let mut cf_opts = rocksdb::Options::default();
                 cf_opts.set_compression_type(rocksdb::DBCompressionType::Lz4);
+                cf_opts.set_block_based_table_factory(block_opts);
```

Comment:
I am curious, where is it documented that setting the block-based factory results in the factory being used for default column families? I would tend to think that setting it globally means that it's used for all CFs.

Reply:
ahh i think i see. It's a sov-schema-db thing not a rocksdb thing. Here we're iterating the tables and creating default config not containing the additional global stuff we've set.

I think we can apply additional configs other than just block based table factory.

Reply:
what other options that are column family specific are you suggesting? all the options we currently set for the db options are global options. I looked into it a bit and couldn't see an option that strike me as potentially useful. There are few fields that I want us to look into related to bloom filtering, but those are also in BlockBasedOptions.